Makefile

```diff
@@ -92,21 +97,26 @@ test/unit:
 
 .PHONY: test/e2e/prow
 test/e2e/prow: export INTEGREATLY_OPERATOR_IMAGE := "registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:integreatly-operator"
-test/e2e/prow: test/e2e
+test/e2e/prow: test/e2e/image
+
+.PHONY: test/e2e/image
+test/e2e/image: export GH_CLIENT_ID := 1234
+test/e2e/image: export GH_CLIENT_SECRET := 1234
+test/e2e/image: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/configmaps cluster/prepare/crd deploy/integreatly-installation-cr.yml
+	operator-sdk --verbose test local ./test/e2e --namespace="$(NAMESPACE)" --go-test-flags "-timeout=60m" --debug --image=$(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: test/e2e
 test/e2e: export GH_CLIENT_ID := 1234
 test/e2e: export GH_CLIENT_SECRET := 1234
 test/e2e: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/configmaps cluster/prepare/crd deploy/integreatly-installation-cr.yml
-	operator-sdk --verbose test local ./test/e2e --namespace="$(NAMESPACE)" --go-test-flags "-timeout=60m" --debug --image=$(INTEGREATLY_OPERATOR_IMAGE)
+	operator-sdk --verbose test local ./test/e2e --namespace="$(NAMESPACE)" --go-test-flags "-timeout=60m" --debug
```

Comment:
@philbrookes @mikenairn Now that the operator is no longer cluster-scoped again, how do you feel about this change to make the default `test/e2e` not require a pre-built image? For the moment, prow will still use the image that it builds.

Reply:
Sorry, only seeing this. Why do we want to do this?

Reply:
Just a thought, so that people running the e2e tests locally don't need to first build the operator image & publish it somewhere. They still could do that though, by using `make test/e2e/image`. Maybe it would be preferable to have the non-image way be `make test/e2e/local` and leave the default be with-image?

It might not be worth the effort though, as I'm not sure how to get the operator logs from the non-image way. I'll remove this change from this PR anyway, as it's just something that could cause a delay in getting it merged.